packages/dev/parcel-namer-docs/package.json

```diff
@@ -4,9 +4,9 @@
   "private": true,
   "main": "DocsNamer.js",
   "engines": {
-    "parcel": "2.0.0-dev.1599"
+    "parcel": "^2.0.0"
   },
   "dependencies": {
-    "@parcel/plugin": "2.0.0-dev.1601"
+    "@parcel/plugin": "2.13.0"
```

Comment:
this should be a range dep

Reply:
hmm why do you think so?

Reply:
well, if someone has updated their own version of parcel in their project, they should still be able to use this without it installing multiple copies
it could probably be `^2.0.0`? unless something has changed or been added in the API that this relies on? but it seems pretty straightforward

Reply:
maybe. I don't think these plugins are published anyway, they are just internal